storm-core/src/clj/backtype/storm/log.clj

```diff
@@ -44,3 +46,11 @@
 (defn log-stream
   [& args]
   (apply log/log-stream args))
+
+(defmacro log-pprint
+  [& args]
+  `(let [^StringWriter writer# (StringWriter.)]
+     (doall
+       (for [object# [~@args]]
+         (pprint object# writer#)))
+     (log-message "\n" writer#)))
```

Comment:
I think it was to clean up the output. Otherwise the data structure would not begin on its own line but instead begin after the timestamp, etc., and so that was not "pretty."